### PR TITLE
Fix orthographic camera rendering for CSS3D on safari on High DPI displays

### PR DESCRIPTION
**Pull request by JohannesDeml**
See https://github.com/mrdoob/three.js/issues/19854

[Demo](https://rawcdn.githack.com/JohannesDeml/three.js/test/css3d-playground/examples/css3d_sandbox_orthographic.html)

### CODE REVIEW

File: examples/jsm/renderers/CSS3DRenderer.js [hunk 1]
```diff
@@ -86,6 +86,7 @@ var CSS3DRenderer = function () {
 	domElement.appendChild( cameraElement );
 
 	var isIE = /Trident/i.test( navigator.userAgent );
+	var isSafari = !! navigator.userAgent.match( /Safari/i ) && ! navigator.userAgent.match( /Chrome/i );
```
Comment (yomotsu) [inline examples/jsm/renderers/CSS3DRenderer.js:89]: How about using test?
`test` is faster than `match` according to https://jsperf.com/test-vs-match-regex/18
```suggestion
	var isSafari = /Safari/.test( navigator.userAgent ) && ! /Chrome/.test( navigator.userAgent );
```

Comment (JohannesDeml) [reply examples/jsm/renderers/CSS3DRenderer.js]: Thanks a lot for the review! Sounds good to me, thanks for pointing it out. I got the isSafari check from the examples (is used in all examples/webgl_gpgpu_*.html files). Maybe we should change the check there as well?

**Comment by mrdoob** [comment]
Thanks!

**Comment by WestLangley** [comment]
With this change, the demo is now broken on my iPhone.

This is not an easy issue.

I would revert this PR until the issue can be fully-understood.

IMG_0459

**Comment by JohannesDeml** [comment]
> 
> 
> With this change, the demo is now broken on my iPhone.

Huh, that's interesting! What phone did you try it on? Maybe I need to take the pixelratio into consideration as well.

But, yep I'm totally with you. I have no clue why it happens, I just try to fight its symptoms.

**Comment by Mugen87** [comment]
> I would revert this PR until the issue can be fully-understood.

I vote for this, too.

**Comment by WestLangley** [comment]
@mrdoob This PR is not correct and needs to be reverted.

**Comment by mrdoob** [comment]
#20067

**Pull request merged**